hazelcast-helper/src/main/java/org/rutebanken/hazelcasthelper/service/HazelCastService.java

```diff
@@ -153,10 +148,30 @@ private HazelcastInstance runHazelcast(final List<String> nodes, String groupNam
         addMgmtIfConfigured(cfg);
 
         cfg.setNetworkConfig(netCfg);
+
+        getAdditionalMapConfigurations().forEach(cfg::addMapConfig);
+
         return Hazelcast.newHazelcastInstance(cfg);
     }
 
     /**
+     * Override this method if you want to provide additional configuration to the map config with name "default"
+     *
+     * @param defaultMapConfig The map config with name "default"
+     */
+    public void updateDefaultMapConfig(MapConfig defaultMapConfig) {}
+
+
+    /**
+     * If you want to provide additional map configurations to hazelcast.
+     *
+     * @return a list with desired map configurations
+     */
+    public List<MapConfig> getAdditionalMapConfigurations() {
```

Comment:
Det er ikke veldig klart for meg hvordan dette skal fungere utifra koden. (Dette trenger ikke å ha noe med koden å gjøre, det kan bare være meg, jeg kjenner ikke hva MapConfig#addMapConfig gjør inne i seg.) 

Min anbefaling vil være å tilføye dette når behovet reiser seg, og det finnes noe kode å peke på som eksempel.